.eslintrc.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
 		"no-case-declarations": "error",
 		"no-var": "warn",
 		"no-multiple-empty-lines": "error",
+		"no-mixed-spaces-and-tabs": ["error","smart-tabs"],
```

Comment:
Please add also

```json
"no-console": "off",
        "no-restricted-syntax": [
            "error",
            {
                "selector": "CallExpression[callee.object.name='console'][callee.property.name!=/^(log|warn|error|info|trace)$/]",
                "message": "Unexpected property on console object was called"
            }
        ]
```

Reply:
ok, but where did you get this "no-restricted-syntax" rule ? i looked at eslint rules and it's different

Reply:
https://eslint.org/docs/latest/rules/no-console#when-not-to-use-it

Reply:
oh i get it, i looked for `no-restricted-syntax` :p, ok thks

Reply:
this part it's done